python/social_bot/envs/cartpole.py

```diff
@@ -111,14 +114,15 @@ def reset(self):
 
 def main():
     env = CartPole()
+    env.reset()
+    env._world.info()
     for _ in range(100):
-        print("reset")
         env.reset()
-        env._world.info()
         total_rewards = 0
         while True:
             obs, reward, done, info = env.step((random.random() - 0.5) * 0.5)
             total_rewards += reward
+            env.render()
```

Comment:
When run out of 100 episodes before terminating, it throws an error on my PC(Ubuntu 16.04, Gazebo 9.6.0):
```
python: /usr/include/boost/thread/pthread/recursive_mutex.hpp:113: void boost::recursive_mutex::lock(): Assertion `!pthread_mutex_lock(&m)' failed.
Aborted (core dumped)
```